ui/apps/platform/src/Containers/AccessControl/Roles/Roles.tsx

```diff
@@ -13,7 +13,6 @@ import {
 } from '@patternfly/react-core';
 
 import NotFoundMessage from 'Components/NotFoundMessage';
-import { getIsDefaultRoleName } from 'constants/accessControl';
```

Comment:
Can you verify that this and the other pull request remove all references to `getIsDefaultRoleName` function, and therefore delete it and `defaultRoles` object from src/constants/accessControl.ts file?